### ISSUE TITLE
Folder colors stop changing after color scheme change on Ubuntu 22.04

### ISSUE THREAD

**Issue by victorkitov** [costales/yaru-colors-folder-color#3]
After installation:
```
sudo add-apt-repository ppa:costales/yaru-colors-folder-color
sudo apt install folder-color yaru-colors-folder-color
```
folder colors and styles can be changed.

However, when I change default Ubuntu 22.04 orange color scheme [Ubuntu settings]->[Appearance]->Style colors (to blue, for example) context menu items remain, however they are unable to change colors of folders (but ability to change icon style - important, in progress, etc. remains).

![image](https://user-images.githubusercontent.com/10587992/216671048-6817469f-37a2-4d3e-8e55-eb6732614320.png)

**Comment by costales** [comment]
Thank you for your feedback! I think is a duplicate of https://github.com/costales/folder-color/issues/21
Anyway, in Ubuntu 22.04 don't install yaru-colors-folder-color

**Comment by victorkitov** [comment]
It works on Ubuntu 22.04 but only with the default (orange) color scheme. Effects disappear only after color change. Maybe it has something to do with the change of icons folder under different color scheme...

**Comment by costales** [comment]
Hi, I'm sure is because of this https://github.com/costales/folder-color/issues/21
`because Yaru-blue inherits from Yaru` :)
